src/entities/order.entity.ts

```diff
@@ -0,0 +1,60 @@
+import { Column, Entity, ManyToOne } from 'typeorm';
+import {
+  EntityPropertyLength,
+  OrderStatus,
+  PaymentStatus,
+  PaymentType,
+} from '../constants';
+import { OrderType } from '../constants/order';
+import { UserEntity } from './user.entity';
+import { StoreEntity } from './store.entity';
+import { BaseEntity } from './base.entity';
+
+@Entity('order')
+export class OrderEntity extends BaseEntity {
+  @Column({ type: 'enum', enum: OrderStatus, default: OrderStatus.PENDING })
+  status: OrderStatus;
+
+  @Column({
+    name: 'payment_status',
+    type: 'enum',
+    enum: PaymentStatus,
+    default: PaymentStatus.INCOMPLETE,
+  })
+  paymentStatus: PaymentStatus;
+
+  @Column({
+    name: 'payment_type',
+    type: 'enum',
+    enum: PaymentType,
+    default: PaymentType.COD,
+  })
+  paymentType: PaymentType;
+
+  @Column({
+    name: 'delivery_address',
+    nullable: true,
+    length: EntityPropertyLength.EXTRA_LARGE,
+  })
+  deliveryAddress: string;
+
+  @Column({
+    name: 'order_type',
+    type: 'enum',
+    enum: OrderType,
+    default: OrderType.PICK_UP,
+  })
+  orderType: OrderType;
+
+  @Column({ default: 0 })
+  value: number;
```

Comment:
trường value này để làm gì nhể?

Reply:
Dạ lúc đầu định trường này là giá trị đơn hàng nhưng mà giá trị này có thể tính thông qua bảng order product và trường shipping_fee nên có thể bỏ trường value này cũng được ạ

Reply:
e giữ lại trường này nha, tránh việc mỗi lần hiển thị thì phải query tính toán từ DB mất tg => đổi tên trường `total` cho dễ hiểu e nhen

Reply:
Dạ vâng ạ